src/fsharp/TypeChecker.fs

```diff
@@ -12074,10 +12077,13 @@ let TcTyconMemberSpecs cenv env containerInfo declKind tpenv (augSpfn: SynMember
 
 let TcModuleOrNamespaceLidAndPermitAutoResolve tcSink env amap (longId : Ident list) =
     let ad = env.eAccessRights
-    let m = longId |> List.map (fun id -> id.idRange) |> List.reduce unionRanges
```

Comment:
This looks good, no need to map when longId is empty

Reply:
yep